### PR TITLE
ITS3 code update

### PR DESCRIPTION
**Pull request by msuljic**
This contains the ITS3 development over the past year squashed into a few commits.

It contains:
- minor updates python bindings
- ITS3 specific producers and converters
- Online Monitor updates needed to run ITS3 converters
- README updates
- ITS3 auxiliary scripts

### CODE REVIEW

**Review by lhuth — COMMENTED** [review]
I have not reviewed the parts in `user/ITS3`. The rest has been viewed. A single comment inline

File: monitors/onlinemon/src/OnlineMon.cxx [hunk 1]
```diff
@@ -236,6 +239,16 @@ void RootMonitor::DoReceive(eudaq::EventSP evsp) {
               continue;
             }
           }
+          if (simpPlane.is_APTS){
```
[the remark this answers is not in the document]
Comment (msuljic) [reply monitors/onlinemon/src/OnlineMon.cxx:242]: The converter was conceptually intended to be fully analog, so I think these lines were added following the example of those just above. If you think it is not the right place to have those lines I can remove them. For consistency maybe the above lines should be removed then as well?

[the remark this answers is not in the document]
Comment (lhuth) [reply monitors/onlinemon/src/OnlineMon.cxx:242]: You are right - I was referring to the two if statements (lines 242-251). 

If you create a hit for each pixel and discriminate later, this is a valid approach on this and can stay unchanged.

[the remark this answers is not in the document]
Comment (lhuth) [reply monitors/onlinemon/src/OnlineMon.cxx:242]: No action required.

Lines 230-241 are legacy that I will not touch for now

**Review by lhuth — APPROVED** [review]

**Pull request merged**